### PR TITLE
CAP-52 Import 3 more fields from the API

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Added 3 new fields (hidden on form and display until designs are available)
- Changed the importer to populate 4 fields
- Changed the importer to not create revisions. this appeared to be causing some issues and I hope that it will resolve some of them.

# Need Review By (Date)
- 9/21

# Urgency
- low

# Steps to Test
1. checkout this branch
1. import config
1. import profiles (i've been using org code `QYIS`)
1. load one of the profiles with devel and see the populated data.

# See Also
- https://github.com/SU-SWS/stanford_person/pull/110
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: config/sync/migrate_plus.migration.su_stanford_person.yml [hunk 1]
```diff
@@ -219,9 +240,27 @@ process:
...
+  su_person_scholarly_interests/value:
+    -
+      plugin: null_coalesce
+      source:
+        - scholarly_interests_fulltext
+        - scholarly_interests_tersetext
 destination:
-  plugin: 'entity_reference_revisions:node'
-  new_revisions: true
-  force_revision: true
```
Comment (sherakama) [inline config/sync/migrate_plus.migration.su_stanford_person.yml:225]: Do these default to false?

**Review by sherakama — COMMENTED** [review]
Code seems fine.
Add tests to validate fields exist please.

**Comment by pookmish** [comment]
Yeah I wasn't sure what kinda tests to put together since they aren't visible anywhere. I'll check for them on the manage fields page for now

**Comment by pookmish** [comment]
tests added

File: config/sync/migrate_plus.migration.su_stanford_person.yml (hunk 1 above)
Comment (pookmish) [reply config/sync/migrate_plus.migration.su_stanford_person.yml:225]: Those are options only with the `entity_reference_revisions` plugin. the regular `entity` doesn't accept additional options, But basically, yes by default it does not create new revisions by using `entity` plugin.

**Review by sherakama — APPROVED** [review]
GTG

**Pull request merged**